spec/view_models/auction_status/expiring_view_model_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe AuctionStatus::ExpiringViewModel do
```

Comment:
I put all of these `AuctionStatus` view models in that module as a way of namespacing them. I do think the null bid presenter should be named `NullBidPresenter`